content/source/docs/cloud/users-teams-organizations/single-sign-on/azure.html.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: "cloud"
+page_title: "Azure - Singlge Sign-On - Terraform Cloud and Terraform Enterprise"
+---
+
+-> **Note:** Single sign-on is a paid feature, available as part of the **Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+# Single Sign-on: Azure
+
+The Azure SSO integration currently supports the following SAML features:
+
+- Service Provider (SP)-initiated SSO
+- Identity Provider (IdP)-initiated SSO
+- Just-in-Time Provisioning
+
+## Configuration (Azure ) Part 1:a
```

Comment:
I'll get @nfagerlund's input here, but I feel that this approach using parts doesn't map to the way we do things elsewhere in the documentation, unless it is a new convention I'm unaware of.